src/status_im/ui/screens/chat/message/link_preview.cljs

```diff
@@ -19,9 +19,14 @@
     (string/starts-with? link (str "https://www." domain)) true
     :else                                                  false))
 
+;;siddarthkay : todo add code for previews here once previews are fixed in new messaging ui
```

Comment:
This TODO could be removed and a Github issue created instead.

Reply:
will do :)

Reply:
here is the issue to track broke community link previews : https://github.com/status-im/status-mobile/issues/14689